src/include/migraphx/onnx.hpp

```diff
@@ -15,6 +15,7 @@ struct onnx_options
     /// Explicitly specify the dims of an input
     std::unordered_map<std::string, std::vector<std::size_t>> map_input_dims = {};
     /// Continue parsing onnx file if an unknown operator is found
+    int64_t max_iter_num        = 10;
```

Comment:
The comment above this applies to `skip_unknown_operators`. Also, please add a comment for this attribute.

Reply:
fixed